exercise/templates/exercise/staff/results.html

```diff
@@ -10,45 +11,175 @@
+                {% trans "Students with submissions" %}
+            </label>
+
+            <label style="margin-left: 10px;" class="checkbox-inline" data-toggle="tooltip"
+                title="{% trans 'Number of students that have received maximum points from the exercise.' %}">
+                <input type="checkbox" class="total-stu-max-checkbox" value="total-stu-max">{% trans "Students with max points" %}
+            </label>
+
+            <label style="margin-left: 10px;" class="checkbox-inline" data-toggle="tooltip"
+                title="{% trans 'Average points received for the exercise per student. Only accounts for students with one or more submissions.' %}">
+                <input type="checkbox" class="avg-p-checkbox" value="avg-p">{% trans "Average points per student" %}
+            </label>
+
+            <label style="margin-left: 10px;" class="checkbox-inline" data-toggle="tooltip"
+                title="{% trans 'Maximum points for the exercise.' %}">
+                <input type="checkbox" class="max-p-checkbox" value="max-p">{% trans "Max points" %}
+            </label>
+        </div>
+
+
+
+        <!--Tags-->
+        <div style="margin: 10px 10px 10px 20px;" class="filter-users">
+            <small>{% trans "Filter users" %}:</small>
+            {% for tag in tags %}
+            <button class="btn btn-default btn-xs" style="background-color:{{ tag.color }};color:{{ tag.font_color }};" data-tag-slug="{{ tag.slug }}">
+              <span class="glyphicon glyphicon-unchecked" aria-hidden="true"></span>
+              {{ tag.name }}
+            </button>
+            {% endfor %}
+        </div>
+
+
+        <!--Navigation bar to choose how data is shown-->
+        <ul style="margin: 10px 0px 50px 0px;" id="exercise-nav-tabs" class="nav nav-tabs">
+            <li id="difficulty-exercises" role="presentation" class="active">
+                <a onclick="createPointTable('difficulty');" data-toggle="tab" href="#">{% trans "Show exercises by difficulty" %}</a>
+            </li>
+            <li id="module-exercises" role="presentation">
+                <a onclick="createPointTable('module');" data-toggle="tab" href="#">{% trans "Show exercises by module" %}</a>
+            </li>
+            <li id="all-exercises" role="presentation">
+                <a onclick="createPointTable('all');" data-toggle="tab" href="#">{% trans "Show all exercises" %}</a>
+            </li>
+        </ul>
+
+        <!--Dropdown button for table exports. The buttons are added inside results_staff.js-->
+        <div class="dropdown d-inline-block">
+            <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">{% trans "Export Data" %}
+                  <span class="caret"></span></button>
+            <ul id="export-button-menu" class="dropdown-menu">
+
+            </ul>
+        </div>
+
+    </div>
+
+        <!--Data table-->
```

Comment:
Indentation starting from this line, doesn't seem to be consistent. Are there mix of tabs and spaces? I prefer tabs fro html files btw.

Reply:
Should be all tabs now the whole file, please note I use vs code for indentation that has its own customizations available. Feel free to check that all works in some other ide.